packages/frontend/core/src/pages/workspace/detail-page/detail-page.tsx

```diff
@@ -301,6 +299,14 @@ export const DetailPage = ({ pageId }: { pageId: string }): ReactElement => {
     currentWorkspace.setPriorityRule(id => id.endsWith(pageId));
   }, [pageId, currentWorkspace]);
 
+  const jumpOnce = useLiveData(pageRecord?.meta.map(meta => meta.jumpOnce));
```

Comment:
I think the `jumpOnce` flag can be moved to PageRecord now since it is a side effect transient flag and not meant to be synced or persisted

Reply:
I feel jumpOnce is a bad design and I want to remove it in the future

Reply:
Agreed.